target_info/target_info.lua

```diff
@@ -0,0 +1,82 @@
+local math = require('math')
+local string = require('string')
+local ui = require('ui')
+local target = require('target')
+
+local text_color = ui.color.lightgrey
+local config_state = {
+    title = 'Target Info',
+    style = 'chromeless',
+    x = 100,
+    y = 0,
+    width = 130,
+    height = 48,
+    color = ui.color.black,
+}
+function math.round(num, prec)
+    local mult = 10^(prec or 0)
+    return math.floor((num * mult + 0.5) / mult)
```

Comment:
The `mult` probably needs to be outside? Otherwise you won't need get any sub-decimal precision.

Reply:
I borrowed that from the windower 4 maths.lua library just saying :P